graylog-storage-elasticsearch7/src/main/java/org/graylog/storage/elasticsearch7/Elasticsearch7Module.java

```diff
@@ -60,6 +61,7 @@ protected void configure() {
                 .to(V20170607164210_MigrateReopenedIndicesToAliasesClusterStateES7.class);
         bindForSupportedVersion(V20200730000000_AddGl2MessageIdFieldAliasForEvents.ElasticsearchAdapter.class)
                 .to(V20200730000000_AddGl2MessageIdFieldAliasForEventsES7.class);
+        bindForSupportedVersion(LuceneQueryParser.class).to(LuceneQueryParser.class);
```

Comment:
Why is this not bound in the ES6 Version?

Reply:
Should be removed in ES7 module as well, it's a forgotten mapping from the time when the lucene query parser was specific for each ES version.